lib/course_planner.ex

```diff
@@ -1,20 +1,25 @@
 defmodule CoursePlanner do
+  @moduledoc """
+  This is the main module of the app
+  """
   use Application
+  alias CoursePlanner.Endpoint, as: Endpoint
+  alias CoursePlanner.Repo, as: Repo
```

Comment:
you can shorten it with `alias CoursePlanner.{Endpoint, Repo}`
also, `alias CoursePlanner.Endpoint` and `CoursePlanner.Endpoint, as: Endpoint` is equivalent.